python/lsst/daf/butler/core/deferredDatasetHandle.py

```diff
@@ -112,3 +116,37 @@ def get(self, parameters: typing.Union[None, dict] = None, **kwargs: dict) -> ty
             mergedParameters = {}
 
         return self._get(mergedParameters)
+
+    def getUri(self, parameters: typing.Union[None, dict] = None, **kwargs: dict) -> typing.Any:
+        """ Retrieves the dataset URI pointed to by this handle
+
+        This handle may be used multiple times, possibly with different
+        parameters.
+
+        Parameters
+        ----------
+        parameters : `dict` or None
```

Comment:
I mentioned this on the commit itself but not in the PR, but getUri doesn't take parameters so I'm confused by the use of the term here. (also the docstring talks about `get`).